components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -80,16 +86,29 @@ class ExperimentsTest {
         `when`(experiments.getExperimentsUpdater(context)).thenReturn(experimentsUpdater)
         experimentSource = source
         `when`(experimentsUpdater.getExperimentSource(configuration)).thenReturn(experimentSource)
+
+        // Since WorkManager doesn't support working with robolectric, need to mock running the task
+        // with a way to execute synchronously to avoid a lot of headaches, but also supporting
```

Comment:
Super nit: "... headaches. For some test cases we also ..."?
The full sentence was hard to parse for me.